src/api/app/models/token/release.rb

```diff
@@ -7,16 +7,20 @@ def self.token_name
   end
 
   def call(_options)
+    # release from remote is not supported
+    raise ActiveRecord::RecordNotFound if @token.package_from_association_or_params.nil?
     manual_release_targets = package_from_association_or_params.project.release_targets.where(trigger: 'manual')
     raise NoReleaseTargetFound, "#{package_from_association_or_params.project} has no release targets that are triggered manually" unless manual_release_targets.any?
 
     manual_release_targets.each do |release_target|
-      release_package(package_from_association_or_params,
+      opts = { filter_source_repository: release_target.repository,
+                        manual: true,
+                        comment: 'Releasing via trigger event' }
+      opts[:multibuild_container] = package_name.gsub(/.*:/, '') if package_name.include?(':')
```

Comment:
So far we did not support multibuild for release tokens. We used `follow_multibuild: false` for release tokens. See [here](https://github.com/openSUSE/open-build-service/blob/87ff5c460f00f0bc8aecd62f644c7a86f6ec19fc/src/api/app/controllers/trigger_controller.rb#L92-L101).  Which means we did not get past instantiating the package.

```ruby
Package.get_by_project_and_name('OBS:Server:Unstable', 'obs-server:obs-bundled-gems', { follow_multibuild: false })
Trying to find local project OBS:Server, remote_project Unstable
Trying to find local project OBS, remote_project Server:Unstable
Package::Errors::UnknownObjectError: Package not found: OBS:Server:Unstable/obs-server:obs-bundled-gems
from /srv/www/obs/api/app/models/package.rb:193:in `get_by_project_and_name'
```

You want to change this?

Reply:
it is support in our internal instance where it is crutial. There is a pull request for this in github since
a few month as well:

  https://github.com/openSUSE/open-build-service/pull/10388

without this all media would get released always no matter which multibuild container was specified.

yes, indeed that parameter needs to change as well.